src/main/java/de/refactoringbot/rest/RefactoringController.java

```diff
@@ -75,40 +52,26 @@
 	@ApiOperation(value = "Perform refactorings with Pull-Request-Comments.")
 	public ResponseEntity<?> refactorWithComments(@PathVariable Long configID) {
 
-		// Create empty list of refactored Issues
```

Comment:
the whole block of code should be moved into the service layer, to the controller just as to call one entry point and handles the exceptions.